packages/dds/tree/src/codec/codec.ts

```diff
@@ -333,7 +333,23 @@ export function withSchemaValidation<
  * Versions of Fluid Framework client packages.
  * @remarks
  * Used to express compatibility requirements by indicating the oldest version with which compatibility must be maintained.
+ *
+ * When no compatibility-impacting change landed in a given version, the value associated with its enum entry may point to the older version which it's fully compatible with.
```

Comment:
Nit: I'm not sure how universally understood "landed" is as a term for "shipped" / "is made". Might be better to say "is made" instead? (100% nitpick)
```suggestion
 * When no compatibility-impacting change is made in a given version, the value associated with its enum entry may point to the older version which it's fully compatible with.
```